assets/global.js

```diff
@@ -533,9 +533,9 @@ class SliderComponent extends HTMLElement {
 
   initPages() {
     this.sliderItemsToShow = Array.from(this.sliderItems).filter(element => element.clientWidth > 0);
-    this.sliderLastItem = this.sliderItemsToShow[this.sliderItemsToShow.length - 1];
-    if (this.sliderItemsToShow.length === 0) return;
-    this.slidesPerPage = Math.floor(this.slider.clientWidth / this.sliderItemsToShow[0].clientWidth);
+    if (this.sliderItemsToShow.length < 2) return;
+    this.sliderItemOffset = this.sliderItemsToShow[1].offsetLeft - this.sliderItemsToShow[0].offsetLeft;
+    this.slidesPerPage = Math.floor(this.slider.clientWidth / this.sliderItemOffset);
```

Comment:
I'm actually now thinking that the total amount can be managed with some liquid logic 🤔 
Sofia added a setting to set amount of products per row for desktop layout and is about to add it on mobile as well. So I'm thinking that we can most likely do something like this: 
```liquid
{{ numberOfItems | minus: amountOfRows | plus: 1 }}
```

And have to spans, one `totalDesktop` and one `totalMobile` so that we can manage visibility via CSS and media queries 🤔  What do you think ?